FHEM/00_SIGNALduino.pm

```diff
@@ -3150,18 +3150,18 @@ sub SIGNALduino_Attr(@) {
   ## Change rfmode
   elsif ($aName eq 'rfmode')          # change receive mode
   {
-    if (not grep /$aVal/, @rfmode) {
-      $hash->{logMethod}->($name, 1, "$name: Attr, $aName $aVal is not supported");
-      return 'ERROR: The rfmode is not supported';
-    }
-
-    if ($init_done) {
-      my $ret = main::SIGNALduino_Attr_rfmode($hash,$aVal);
-
-      if (defined $ret) {
-        return $ret;
-      } else {
-        $hash->{logMethod}->($name, 3, "$name: Attr, $aName switched to $aVal");
+    if( $cmd eq 'set' ) {
+      if (not grep /$aVal/, @rfmode) {
```

Comment:
Ich weiss, dass Du es nur kopiert hast.
Mir scheint aber, dass wir hier doch eigentlich ein fullmatch brauchen und kein Teilmatch.

Reply:
Du meinst sicher die Zeile?
`if (not grep /$aVal/, @rfmode) {`
Die habe ich eigentlich nicht angerührt. Als "neu" wird diese sicher nur gekennzeichnet, weil sich der Einzug geändert hat.

Die sollte dann wahrscheinlich besser so ausehen?
`if (not grep /^$aVal$/, @rfmode) {`

Reply:
Ich hab es mit first und einem vergleichsoperator angepasst. Regex brauchen wir ja eigentlich nicht schätze ich.